pkgs/applications/science/math/sage/threejs-sage.nix

```diff
@@ -12,7 +12,8 @@ stdenv.mkDerivation rec {
   };
 
   installPhase = ''
-    mkdir -p $out/lib/node_modules/three
-    cp -r build version $out/lib/node_modules/three
+    mkdir -p "$out/lib/node_modules/three/"
+    cp version "$out/lib/node_modules/three"
+    cp -r build "$out/lib/node_modules/three/$(cat version)"
```

Comment:
Why not use install?

Reply:
That's a good suggestion. Since this is already merged, I will do this change in the forthcoming Sage 9.5 PR.